csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+We assume that they have trouble managing their sleep due to demands from their studies.
+In order to truly improve their sleep habits, they want to know how much sleep they really need and what the flow on effects of a good or poor quality sleep are.
+
+Answering these simple questions has already helped us to narrow down the scope of the problem and gives us a starting point for our research.
+What we think our users want and what they actually want are often two different things.
+Therefore it’s important to get some user input early on so we can better understand our users wants and needs.
+We are going to start involving the users by creating a simple survey.
+
+First, we decide on what we actually want to learn from our users:
+
+- The “why” behind their current behaviour,
+
+- their frustrations,
+
+- where they want to improve, and
+
+- what would help make this process easier.
+
+However, these general questions would be difficult for our target users to answer, so we translate them into more specific questions:
+
+- *On average, how many hours of sleep do you get each night?*
+
+- *Do you usually feel rested when you wake up?*
+
+- *Do you wake up naturally most mornings or do you use an alarm clock to wake you?*
+
+- *Describe for me what your bedtime routine looks like.*
+
+With only these four questions we can already start to determine the users current behaviour, what influences it, as well as their frustrations.
+A comprehensive survey will give us a much deeper understanding of the user.
+
+{panel type="teacher-note"}
+
+# Survey Tips
+
+A 20 - 30 question survey given to 10 - 20 people should give a useful amount of data for the students to work with.
+
+Remind your students about the ethics of collecting people’s data.
+Their surveys should be anonymous (i.e. don’t ask for names or contact information) as there shouldn’t be any need to tie a response to a particular person.
+
+{panel end}
+
+### Step 2: Identify your competitors
+
+**UX Method: Competitive Analysis**
+
+Thanks to our survey in step 1 we now have a much better idea of our target users and what their needs are.
+The next step is therefore to do some research on what solutions are already out there.
+
+We’ll use what is called a *Competitive Analysis*.
+A *Competitive Analysis* is when you research the existing solutions for your particular problem.
+Key questions you want to answer are:
+
+- What worked for them?
+
+- What didn’t?
+
+- What features do they have in common?
+
+Here’s a very simple example of a competitive analysis table for our sleep app:
+
+|                                         | [Fitbit](https://itunes.apple.com/au/app/fitbit/id462638897?mt=8) | [Pillow](https://itunes.apple.com/au/app/pillow-automatic-sleep-tracker/id878691772?mt=8) | [Sleepzy](https://itunes.apple.com/us/app/sleepzy-sleep-cycle-tracker/id1064910141#?platform=ipad) | [Sleep Better](https://itunes.apple.com/au/app/sleep-better-sleep-tracker/id922541792#?platform=ipad) |
```

Comment:
Is it better to use real world examples which can be outdated quickly (also require an iOS device), or App A, App B, App C, App D? Thoughts? I feel like the names are unimportant, but I may be missing something.

Side note: Pillow is great, I used it for a long time!